apps/storybook/.storybook/main.ts

```diff
@@ -20,6 +21,13 @@ const config: StorybookConfig = {
       include: [resolve(__dirname, '../../../packages/react/**/**.tsx')], // <- This is the important line.
       shouldExtractLiteralValuesFromEnum: true,
       shouldRemoveUndefinedFromOptional: true,
+      propFilter: ({ name, declarations = [] }: PropItem) => {
+        const hasDescription = declarations.some(
+          ({ fileName }) => !fileName.includes('node_modules'),
+        );
+
+        return hasDescription && name !== 'popovertarget'; // Skip popovertarget @types/react-dom patch
+      },
```

Comment:
```suggestion
      propFilter: ({ name }: PropItem) => {
        return name !== 'popovertarget'; // Skip popovertarget @types/react-dom patch
      },
```

Why are we checking description? This may cause some side effects 🤔

Reply:
This is what `react-docgen-typescript` does out of the box, but looses this functionality when providing a custom `propFilter`. This code is from the documentation of `react-docgen-typescript`, and ensures that properties that are described in Typescripts `dom.ts` (which is _all_ the HTML attributes) does not bleed into documentation :)

Reply:
Alright sounds good then, could you add a comment about it in code so we don't delete this later on?
```suggestion
      propFilter: ({ name, declarations = [] }: PropItem) => {
      /* Default from `react-docgen-typescript` we have to add back when using a custom prop filter */
        const hasDescription = declarations.some(
          ({ fileName }) => !fileName.includes('node_modules'),
        );

        return hasDescription && name !== 'popovertarget'; // Skip popovertarget @types/react-dom patch
      },
```